<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionCreate"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionStatusBarVisible"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Information</string>
    </property>
    <addaction name="actionProducts"/>
    <addaction name="actionPerson"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>Query</string>
    </property>
    <addaction name="actionEatenQuery"/>
    <addaction name="actionDateUseProd"/>
    <addaction name="actionFractPFC"/>
   </widget>
   <widget class="QMenu" name="menutools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBodyMassIndex"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuInfo"/>
   <addaction name="menuQuery"/>
   <addaction name="menutools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Tool bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionStatusBarVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionProducts">
   <property name="text">
    <string>Food</string>
   </property>
  </action>
  <action name="actionPerson">
   <property name="text">
    <string>Personal data</string>
   </property>
  </action>
  <action name="actionEatenQuery">
   <property name="text">
    <string>Amount of food eaten</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>test</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
  <action name="actionBodyMassIndex">
   <property name="text">
    <string>Body mass index</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionDateUseProd">
   <property name="text">
    <string>Dates of use of the product</string>
   </property>
  </action>
  <action name="actionFractPFC">
   <property name="text">
    <string>Fractions of proteins, fats and carbohydrates</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
